src/org/javarosa/core/model/instance/TreeElement.java

```diff
@@ -55,8 +55,7 @@
  * An element of a FormInstance.
  *
  * TreeElements represent an XML node in the instance. It may either have a value (e.g., <name>Drew</name>),
- * a number of TreeElement children (e.g., <meta><device /><timestamp /><user_id /></meta>), or neither (e.g.,
- * <empty_node />)
+ * a number of TreeElement (e.g., <meta><device /><timestamp /><user_id /></meta>), or neither (e.g.,<empty_node />)
```

Comment:
“a number of” should be followed by a plural noun.

Reply:
Corrected this one too. Marking resolved.